handler.py

```diff
@@ -145,3 +165,29 @@ def poll_running_jobs(event, context):
     event['finishedJobCount'] = len(event['failedJobIds']) + len(event['succeededJobIds'])
     event['unfinishedJobCount'] = len(event['jobIds']) - event['finishedJobCount']
     return event
+
+
+def finished(event, _context):
+    """Publish details about successfully finished jobs."""
+
+    # first, if there were no succeeded jobs, return instead of publishing details about builds
+    if len(event['succeededJobIds']) < 1:
+        return event
+
+    # fetch all jobs, removing those which are not in our succeeded id list
+    r = batch_client.list_jobs(jobQueue=os.environ['JOB_QUEUE_ARN'], jobStatus='SUCCEEDED')
+    print(f'r: {r}')
+    succeeded_jobs = [i for i in r['jobSummaryList'] if i['jobId'] in event['succeededJobIds']]
```

Comment:
Maybe this is a stupid question, but is there a case when a job status would be `SUCCEEDED` but not in `event['succeededJobIds']`?  Isn't the `r` data structure already purely `SUCCEEDED` jobs?